data-access/mandataris.ts

```diff
@@ -309,29 +307,25 @@ export async function terminateMandataris(
     );
   }
 
-  const statusBeeindigd = sparqlEscapeUri(MANDATARIS_STATUS.BEEINDIGD);
   const datumBeeindigd = sparqlEscapeDateTime(endDate);
   const terminateQuery = `
     PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
 
     DELETE {
       GRAPH ?graph {
       ${sparqlEscapeTermValue(mandataris)}
-        mandaat:status ?status ;
         mandaat:einde ?einde .
       }
     }
     INSERT {
       GRAPH ?graph {
         ${sparqlEscapeTermValue(mandataris)}
-          mandaat:status ${statusBeeindigd} ;
           mandaat:einde ${datumBeeindigd} .
       }
     }
     WHERE {
       GRAPH ?graph {
         ${sparqlEscapeTermValue(mandataris)}
-          mandaat:status ?status ;
           mandaat:einde ?einde .
```

Comment:
I think an optional is missing here, but I fix it in a different PR, so should not really be a problem

Reply:
Is not even used in this PR, so is fine